### ISSUE TITLE
[Accessibility] Loading Info is Not Converyed When Dialog is Open

### ISSUE THREAD

**Issue by zinc0x1E** [MicrosoftDocs/msteams-docs#8577]
# Description

Loading info of message extension action commands should be conveyed by screen reader

# Expected Behaviour

Given a user is using screen reader, when user open a message extension dialog and it is loading, then the loading info should be converyed

# Actual Behaviour

Nothing relevant will be read by screen reader when dialog is loading

# Card Json
```json
NA
```

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this issue, we will check this issue and update you.

**Comment by Meghana-MSFT** [comment]
We are also able to repro this issue, we have raised a bug for the same. We will keep you posted on the updates. Thank you.

**Comment by Prasad-MSFT** [comment]
Update:

The issue is now resolved. Narrator now announces the loading state as expected.

<img width="294" height="296" alt="Image" src="https://github.com/user-attachments/assets/03ab1749-4685-469a-be75-5cefb31d5c4e" />
